onnx/defs/tensor/defs.cc

```diff
@@ -34,33 +34,78 @@ User must be aware of precision loss and value change caused by range difference
 For example, a 64-bit float 3.1415926459 may be round to a 32-bit float 3.141592. Similarly, converting
 an integer 36 to Boolean may produce 1 because we truncate bits which can't be stored in the targeted type.
 
-In more detail, the conversion among numerical types should follow these rules:
+In more detail, the conversion among numerical types should follow these rules
+if the destination type is not a float 8 type.
 
 * Casting from floating point to:
-  * floating point: +/- infinity if OOR (out of range).
+  * floating point: +/- highest value if OOR (out of range),
+    infinities are converting into NaN if infinity is not available
   * fixed point: undefined if OOR.
   * bool: +/- 0.0 to False; all else to True.
 * Casting from fixed point to:
-  * floating point: +/- infinity if OOR. (+ infinity in the case of uint)
+  * floating point: +/- highest value if OOR (out of range),
+    infinities are converting into NaN if infinity is not available
   * fixed point: when OOR, discard higher bits and reinterpret (with respect to two's complement representation for
     signed types). For example, 200 (int16) -> -56 (int8).
   * bool: zero to False; nonzero to True.
 * Casting from bool to:
   * floating point: `{1.0, 0.0}`.
   * fixed point: `{1, 0}`.
   * bool: no change.
+
+Float 8 type were introduced to speed up the training of
+deep models. By default the conversion of a float *x* obeys
+to the following rules. `[x]` means the rounded value.
```

Comment:
We may need to expand this a little, e.g. "the value rounded to the target mantissa width"

Reply:
done